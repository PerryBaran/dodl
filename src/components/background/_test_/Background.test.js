import React from "react";
import { render, screen, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom'
import Backround from "../Background";
import renderer from "react-test-renderer";

afterEach(cleanup);

it("renders without crashing", () => { 
    render(<Backround/>);
});

it("matches snapshot", () => {
    const tree = renderer.create(<Backround/>).toJSON();
    expect(tree).toMatchSnapshot();
});