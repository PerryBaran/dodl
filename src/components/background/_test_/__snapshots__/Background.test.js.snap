// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="container"
>
  <div>
    <span
      style={
        Object {
          "display": "inherit",
          "height": "400px",
          "position": "relative",
          "width": "400px",
        }
      }
    >
      <span
        style={
          Object {
            "animation": "react-spinners-PuffLoader-puff-1, react-spinners-PuffLoader-puff-2",
            "animationDelay": "-1s",
            "animationDuration": "2s",
            "animationFillMode": "both",
            "animationIterationCount": "infinite",
            "animationTimingFunction": "cubic-bezier(0.165, 0.84, 0.44, 1), cubic-bezier(0.3, 0.61, 0.355, 1)",
            "border": "thick solid #df88cb",
            "borderRadius": "50%",
            "height": "400px",
            "left": "0",
            "opacity": "1",
            "position": "absolute",
            "top": "0",
            "width": "400px",
          }
        }
      />
      <span
        style={
          Object {
            "animation": "react-spinners-PuffLoader-puff-1, react-spinners-PuffLoader-puff-2",
            "animationDelay": "0s",
            "animationDuration": "2s",
            "animationFillMode": "both",
            "animationIterationCount": "infinite",
            "animationTimingFunction": "cubic-bezier(0.165, 0.84, 0.44, 1), cubic-bezier(0.3, 0.61, 0.355, 1)",
            "border": "thick solid #df88cb",
            "borderRadius": "50%",
            "height": "400px",
            "left": "0",
            "opacity": "1",
            "position": "absolute",
            "top": "0",
            "width": "400px",
          }
        }
      />
    </span>
  </div>
</div>
`;
